src/hotspot/share/c1/c1_Compiler.cpp

```diff
@@ -167,6 +167,9 @@ bool Compiler::is_intrinsic_supported(vmIntrinsics::ID id) {
   case vmIntrinsics::_dsin:
   case vmIntrinsics::_dcos:
   case vmIntrinsics::_dtan:
+  #if defined(X86)
```

Comment:
Use `#ifdef AMD64` for x64 only

Reply:
Thanks Vladimir! Please see the code updated with `#ifdef AMD64`.